File: 0001-two-sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):  # j starts from i+1 to avoid using same index
                if nums[i] + nums[j] == target:
                    return [i, j]

File: 0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> m=new HashMap<>();
        int l=nums.length;
        for(int i=0;i<l;i++){
            int c=target-nums[i];
            if(m.containsKey(c)){
                return new int[]{m.get(c),i};
            }
            m.put(nums[i],i);
        }
        return new int[]{};
    }
        
        
}

